tools/photofinish/.photofinish.toml

```diff
@@ -0,0 +1,12 @@
+[first-test-scenario]
+files = [
+  "../../test/fixtures/discovery/host/expected_published_host_discovery.json",
+  "../../test/fixtures/discovery/sap_system/sap_system_discovery_application.json",
```

Comment:
As discussed with @rtorrero,  `../../test/fixtures/discovery/sap_system/sap_system_discovery_application.json` cannot be POSTed to the collector.
We're probably missing something like `expected_published_sap_system_discovery_application.json` with `agent_id` and `discovery_type`

Reply:
@nelsonkopliku @rtorrero I believe we need a new the fixtures folders with the acceptance testing scenarios, and then see if those fixtures could be used in the unit tests, maybe even improving them.
So, as for now i would keep the test fixtures as-is, and maybe comment out the toml till we don't have real scenarios to wire-up.

Wdyt?

Reply:
@fabriziosestito yep, agreed, I'm putting something together, I'll ping you if I get lost on the way :-)